Modules/Common/include/Common/TH1Reductor.h

```diff
@@ -24,7 +24,7 @@ namespace o2::quality_control_modules::common
 /// \brief A Reductor which obtains the most popular characteristics of TH1.
 ///
 /// A Reductor which obtains the most popular characteristics of TH1.
-/// It produces a branch in the format: "mean/D:stddev:entries"
+/// It produces a branchin the format: "mean/D:stddev:entries"
```

Comment:
?

Reply:
I think this fix got reverted when I updated my QC yesterday, as I remember fixing it already once. Now it should be fine